src/PositionObserver.php

```diff
@@ -45,6 +45,11 @@ public function saving($model)
         if ($model->isPositionUpdateDisabled() === false) {
             // Get the position for current and old value
             $position = $model->getPosition();
+            
+            // Prevent modifying position column when updating and position columns has not changed
+            if ($model->exists === true && $model->isDirty($model->getPositionColumn()) === false) {
```

Comment:
next time :) `$model->getPositionColumn()` extract to variable

Reply:
Ok 👍